01-filedir.md

```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we now see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). It may also be called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```

Comment:
language pedantry: can you change "now" to something else, like "Finally, we also see..." or something along those lines?  "now" to me implies immediacy, and this is the last thing being discussed after the command/output is given.